content/_partials/video.html.haml

```diff
@@ -0,0 +1,21 @@
+- video_id = rand(1000)
+
+:css
+  .video_#{video_id} {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
+    height: 0;
+  }
+  .video_#{video_id} iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+  #ProjectVideo_#{video_id} {
+    background: #{video_background};
```

Comment:
where is video_background defined?

Reply:
Thanks for catching that mistake.  Removed the reference to video_background, since it is not needed.  

Removed the styling of the ProjectVideo link since it is not needed either